packages/xo-server-test/src/_xoConnection.js

```diff
@@ -278,7 +278,7 @@ afterAll(async () => {
   await xo.close()
   xo = null
 })
-afterEach(() => xo.deleteTempResources())
+afterEach(() => xo.deleteTempResources(), 30e3)
```

Comment:
Please add a comment here to explain why this change.

Reply:
The second `afterEach` param is its timeout, So, IMHO, it's obvious that we add the timeout because it takes more time.

Reply:
Why this value though?

Reply:
As mentioned in the PR description, the `afterEach` duration takes for me `~21ms` i round it to `30ms`.

Reply:
IMHO we need it because if we ask these questions now we will ask the same thing after a while, right?

Reply:
For me it's obvious that we add a timeout because `deleteTempResources` takes time to execute. 

`30ms` is just a choice to start, it can be modified in the future, and it can differs at each execution due to externals dependencies (xen-server response/networking/xo-server response ...).

I need to know the rule, when should we add a comment? In order to not  add a comment on each change because each PR introduce new things to XO.